### PR TITLE
Add style for the main cta section

### LINKED ISSUE #31: Build the mobile app cta section

### PR DESCRIPTION
**Pull request by gmagnenat**
@KingLyrics I had a try at the main cta section. Quite a tricky one.. 
I'm not sure the approach is right. The background pattern should be on the section itself.

### CODE REVIEW

**Comment by KingLyrics** [comment]
Alright, I will go through the code and get back to you as soon as possible.

File: src/index.html [hunk 1]
```diff
@@ -39,18 +39,14 @@ <h1 class="hero__title"> We make your music sound extraordinary.</h1>
...
+	<main class="main">
 		<div class="container">
-			<div class="blue-art">
-				<img src="./assets/bg-pattern-2.svg" alt="">
-			</div>
-
-			<div class="app-img">
-				<img src="./assets/illustration-app.png" alt="">
+			<div class="main__illustration">
+				<img src="./assets/illustration-app.png" alt="Screenshot of the equalizer application showing levels of an equaliser, a decible volume, a selection of tone and an apply settings button">
```
Comment (KingLyrics) [inline src/index.html:45]: So you want to change this so that the black background has a section on it own?

Comment (KingLyrics) [reply src/index.html:45]: Sorry you mean the background pattern?

**Review by KingLyrics — APPROVED** [review]

File: src/index.html (hunk 1 above)
Comment (gmagnenat) [reply src/index.html:45]: The background pattern should be on the whole main section, if you check the design you see it's always top center and behind the content.

File: src/sass/main.scss [hunk 2]
```diff
@@ -150,12 +150,12 @@ body {
...
+	background-size: 74%;
 	background-color: $clr-neutral-800;
 	@include radius-full;
 
 	&__illustration {
-		background: url(../assets/bg-pattern-2.svg) no-repeat top center;
-		background-size: 74%;
 		padding-inline: 3rem;
 		margin-inline: auto;
 		max-height: 380px;
```
Comment (gmagnenat) [inline src/sass/main.scss:161]: @KingLyrics not supper confident with this max-height.. but let's keep it for now

File: src/sass/main.scss [hunk 3]
```diff
@@ -150,12 +150,12 @@ body {
 
 // main section
 .main {
+	background: url(../assets/bg-pattern-2.svg) no-repeat top center;
```
Comment (gmagnenat) [inline src/sass/main.scss:153]: @KingLyrics I moved the background to the whole section

**Pull request merged**